rfcs/feature-set-metadata-file-to-web-platform-tests.md

```diff
@@ -0,0 +1,220 @@
+# RFC #163: Add web_feature metadata file to web-platform-tests
+
+Author: @jcscottiii
+
+# Introduction
+
+The web-platform-tests (WPT) project is a valuable resource for testing web
+platform features and contains valuable metadata such as feature spec links.
+Recently, the [WebDX Community Group](https://www.w3.org/community/webdx/)
+began creating the [web-features](https://github.com/web-platform-dx/web-features)
+repository. That repository serves as a basic shared catalog of feature
+definitions of the web platform. The repository itself doesn't intend to produce
+data but rather link to existing data that will inform audiences which features
+are part of [Baseline](https://web.dev/baseline/). However, there's a great
+opportunity to connect the WPT ecosystem to the web-features catalog. By doing
+this, it would enable users of wpt.fyi to filter by web-features grouping, which
+is similar to
+[the ability to filter by spec links](https://github.com/web-platform-tests/wpt.fyi/issues/1489).
+The RFC proposes the addition of a WEB_FEATURE.yml metadata file which would
+enable the linkage between WPT and web-features.
+
+# Proposed change
+
+The proposed change includes:
+
+1. Introduce a new metadata file, WEB_FEATURE.yml
```

Comment:
I think maybe there was a miscommunication here (or I just changed my mind), but using the existing META.yml files is fine in my opinion. I'm just not keen for this to end up affecting the existing `MANIFEST.json` code. _But_ maybe a new file works better for the use case of not creating reviews for changes. So this is also fine.

Reply:
It was the only way I could think of to get around the unnecessary review pings issue. ([comment](https://github.com/web-platform-tests/rfcs/pull/157#discussion_r1268017940)) (aside from having wpt-pr-bot read the file directly before assigning reviewers.